### DISCUSSION TITLE
Authentication for Console

### DISCUSSION THREAD

**Discussion by Faeez-Ali** [akvorado/akvorado#559] [Q&A]
Hey @vincentbernat ! hope you are doing good, How can we add authentication to console, kindly guide me I have read documentation and tried but I am unable to do so.

**Comment by vincentbernat** [comment]
I don't have much to add compared to what is in the documentation: https://demo.akvorado.net/docs/configuration#authentication.

**Comment by echo1377** [comment]
Try to add authentication through traefik 

add in docker-compose.yml
akvorado-console:
labels:
 - "traefik.http.routers.akvorado-console.middlewares=frontend-auth"
 - "traefik.http.routers.akvorado-console.middlewares=auth"
 - "traefik.http.middlewares.auth.basicauth.users=**USERNAME:PASSWORDHASH**"
 
manual:
https://doc.traefik.io/traefik/middlewares/http/basicauth/

**Reply by Faeez-Ali** [reply]
hey @echo1338, thanks for the suggestion! should I have to also mention Akvorado-console port (i.e 8083) any where or any thing else to configure , or should i have to only configure labels in docker-compose.yml of Traefik.

**Reply by echo1377** [reply]
Only labels.
